src/components/Home/Home.js

```diff
@@ -0,0 +1,54 @@
+import { brands, categories } from "../../data";
+import { Link } from "react-router-dom";
+import "./home.css";
+
+export default function Home() {
+  return (
+    <>
+      <div className="home-banner-container">
+        {/* <img src={Banner} className="home-banner-img" /> */}
+        <div className="home-banner-content">
+          <h2>Hello Devs 👋</h2>
```

Comment:
Emoji's should be wrapped in a span tag with role img like this- <span role="img" aria-label="hello">👋</span>